frontend/src/components/common/form/Typeahead.tsx

```diff
@@ -70,7 +70,8 @@ export function TypeaheadField<T extends TypeaheadModel>({
     setFieldTouched(name, true);
     if (
       (!getIn(name, 'value') && val !== '' && getIn(values, name) === '') ||
-      getIn(values, name) === null
```

Comment:
There was another bug with selecting closest match on subsequent match where the value was undefined rather than `''` or `null` causing the closest match not to fire. This solves that